test/functional/tests/data_integrity/pattern_utils.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import random
+from enum import Enum
+
+
+def _get_seq_pattern():
+    seq_pattern = "0x"
+    for i in range(0, 256):
+        seq_pattern += f"{hex(i).lstrip('0x'):0>2}"
+    return seq_pattern.strip()
+
+
+def _get_random_pattern():
+    random_pattern = "0x"
+    for i in random.sample(range(256), random.randint(1, 256)):
+        random_pattern += f"{hex(i).lstrip('0x'):0>2}"
+    return random_pattern.strip()
+
+
+class Patterns(Enum):
+
+    def __str__(self):
+        return self.value
+
+    cyclic_pattern = "0x00336699ccffcc996633"
+    seq_pattern = _get_seq_pattern()
+    high_pattern = "0xaa"
+    low_pattern = "0x84210"
+    zeroes_pattern = "0x00"
+    ones_pattern = "0xff"
+    bin_1_pattern = high_pattern
+    bin_2_pattern = "0x55"
+    random_pattern = _get_random_pattern()
```

Comment:
remove '_pattern' from names (so we can call Pattern.cyclic instead of Patterns.cyclic_pattern)